.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!-- 
+Please use this template when submitting a new feature request with as much details as possible. Not doing so may result in the issue not being addressed in a timely manner or eventually closed.
+-->
+# Title
+<!-- 
+The same title used to create the issue (optional)
+-->
+
+## Summary 
+
+<!-- 
+A quick summary about the feature request or bug fix with issue number. Ex: Add a new logging system described in #<issue number> 
+-->
+
+## Type
+
+[] Bug fix
+[] Feature request
+[] Enhancement
```

Comment:
Is it worth having a separate category for docs?

Reply:
@JoelSpeed updated now, please let me know if that works for you.